warehouse/integrations/clickhouse/clickhouse.go

```diff
@@ -1032,9 +1034,25 @@ func (ch *Clickhouse) LoadUserTables(ctx context.Context) (errorMap map[string]e
 	return
 }
 
-func (ch *Clickhouse) LoadTable(ctx context.Context, tableName string) error {
-	err := ch.loadTable(ctx, tableName, ch.Uploader.GetTableSchemaInUpload(tableName))
-	return err
+func (ch *Clickhouse) LoadTable(ctx context.Context, tableName string) (*types.LoadTableStats, error) {
+	preLoadTableCount, err := ch.totalCountIntable(ctx, tableName)
+	if err != nil {
+		return nil, fmt.Errorf("pre load table count: %w", err)
+	}
+
+	err = ch.loadTable(ctx, tableName, ch.Uploader.GetTableSchemaInUpload(tableName))
+	if err != nil {
+		return nil, fmt.Errorf("loading table: %w", err)
+	}
+
+	postLoadTableCount, err := ch.totalCountIntable(ctx, tableName)
+	if err != nil {
+		return nil, fmt.Errorf("post load table count: %w", err)
+	}
```

Comment:
We can probably have flags later to disable it in case it's taking too long to execute. 
Currently, checked for existing customers and don't see any timeout error logs corresponding to `total count in table`.